styles/dark-serif.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schemas.wp.org/wp/6.2/theme.json",
```

Comment:
This can probably the 6.3 schema

Reply:
ok. i solved it

Reply:
I checked twentytwentythree using their trunk schema. Can I Updating schema from version 6.2 to trunk